bridges/NovayaGazetaEuropeBridge.php

```diff
@@ -0,0 +1,119 @@
+<?php
+class NovayaGazetaEuropeBridge extends BridgeAbstract
+{
+
+	const MAINTAINER = 'sqrtminusone';
+	const NAME = 'Novaya Gazeta Europe Bridge';
+	const URI = 'https://novayagazeta.eu';
+
+	const CACHE_TIMEOUT = 3600; // 1 hour
+	const DESCRIPTION = 'Returns articles from Novaya Gazeta Europe';
+
+	const PARAMETERS = array(
+		'' => array(
+			'language' => array(
+				'name' => 'Language',
+				'type' => 'list',
+				'defaultValue' => 'ru',
+				'values' => array(
+					'Russian' => 'ru',
+					'English' => 'en',
+				)
+			)
+		)
+	);
+
+	public function collectData()
+	{
+		$url = 'https://novayagazeta.eu/api/v1/get/main';
+		if ($this->getInput('language') != 'ru') {
+			$url .= '?lang=' . $this->getInput('language');
+		}
+
+		$json = getContents($url);
+		$data = json_decode($json);
+
+		foreach ($data->records as $record) {
+			foreach ($record->blocks as $block) {
+				if (!property_exists($block, 'date')) {
+					continue;
+				}
+				$body = '';
+				if (property_exists($block, 'body') && $block->body !== null) {
+					$body = self::convertBody($block->body, $block->lead);
+				} else {
+					$record_json = getContents("https://novayagazeta.eu/api/v1/get/record?slug={$block->slug}");
+					$record_data = json_decode($record_json);
+					$body = self::convertBody($record_data->record->body, $record_data->record->lead);
+				}
+				$title = strip_tags($block->title);
+				if (property_exists($block, 'subtitle') && mb_strlen($block->subtitle) > 0) {
```

Comment:
This is occasionally null, resulting in warnings:
` Deprecated: mb_strlen(): Passing null to parameter #1 ($string) of type string is deprecated in bridges/NovayaGazetaEuropeBridge.php on line 50`

Does this work: `!empty($block->subtitle)`?

Reply:
Yes, `empty` does work. Also, it seems like that key is actually always defined in the API.